intranet/apps/eighth/views/admin/attendance.py

```diff
@@ -36,13 +37,13 @@ def delinquent_students_view(request):
     start_date = request.GET.get("start", "")
     end_date = request.GET.get("end", "")
 
-    if not lower_absence_limit.isdigit():
+    if any(c not in string.digits for c in lower_absence_limit):
```

Comment:
Might be worth to factor the digit check into a `util` method since you repeat this so often.